learn/safe-core/safe-core-account-abstraction-sdk/relay-kit.md

```diff
@@ -13,47 +15,145 @@ The [Relay Kit](https://github.com/safe-global/account-abstraction-sdk/tree/main
+) to send 0.0005 BNB and submits it to Gelato
+1. [Track the relay request](https://docs.gelato.network/developer-services/relay/quick-start/tracking-your-relay-request) of [Gelato Task ID 0x1bf7](https://relay.gelato.digital/tasks/status/0x1bf7664a1e176472f604bb3840d3d2a5bf56f98b60307961c3f8cee099f1eeb8)
+1. [Transaction 0x814d3](https://bscscan.com/tx/0x814d385c0ec036be65663b5fbfb0d8d4e0d35af395d4d96b13f2cafaf43138f9) executed on the blockchain
+
+### Deposit Polygon USDC into Gelato 1Balance
+
+See: [How Can I use 1Balance?](https://docs.gelato.network/developer-services/relay/payment-and-fees/1balance#how-can-i-use-1balance)
+
+### Import Packages
 
 ```typescript
+import { ethers } from 'ethers'
 import { GelatoRelayAdapter, MetaTransactionOptions } from '@safe-global/relay-kit'
+import Safe from '@safe-global/safe-core-sdk'
+import { MetaTransactionData, OperationType } from '@safe-global/safe-core-sdk-types'
+```
+### Initialize your Transaction Settings
 
-const relayAdapter = new GelatoRelayAdapter(GELATO_RELAY_API_KEY)
+Modify the variables to customize to match your desired transaction settings.
 
+```typescript
+// https://chainlist.org
+const RPC_URL='https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161'
+const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
+const signer = new ethers.Wallet(process.env.OWNER_1_PRIVATE_KEY!, provider)
+const safeAddress = '0x...' // Safe from which the transaction will be sent
+const chainId = 5
+
+// Any address can be used for destination. In this example, we use vitalik.eth
+const destinationAddress = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+const withdrawAmount = ethers.utils.parseUnits('0.005', 'ether').toString()
+
+// Usually a limit of 21000 is used but for smart contract interactions, you can increase to 100000 because of the more complex interactions.
+const gasLimit = '100000'
+```
+
+### Create your Transaction Object
+
+```typescript
+// Create a transaction object
+const safeTransaction: MetaTransactionData = {
+  to: destinationAddress,
+  data: '0x',// leave blank for ETH transfers
+  value: withdrawAmount,
+  operation: OperationType.Call
+}
 const options: MetaTransactionOptions = {
-  isSponsored: true // This parameter is mandatory to use the 1Balance method
+  gasLimit: ethers.BigNumber.from(gasLimit),
+  isSponsored: true
+}
+```
+
+### Create the Protocol and Relay Adapter Instance
+
+```typescript
+const ethAdapter = new EthersAdapter({
+  ethers,
+  signerOrProvider: signer
+})
+
+const safeSDK = await Safe.create({
+  ethAdapter,
+  safeAddress
+})
+
+const relayAdapter = new GelatoRelayAdapter(process.env.GELATO_RELAY_API_KEY!)
+```
+
+### Prepare the Transaction
+
+```typescript
+const standarizedSafeTx = await safeSDK.createTransaction({
+  safeTransactionData: safeTransaction
+})
```

Comment:
I would say that the standardization of a tx is something that happens internally.
Also using `safeTransactionData` here
```suggestion
const safeTransaction = await safeSDK.createTransaction({ safeTransactionData })
```

Reply:
Can you elaborate on what you mean by "standardization of a tx is something that happens internally"?